src/core/lib/gprpp/orphanable.h

```diff
@@ -72,7 +72,7 @@ inline OrphanablePtr<T> MakeOrphanable(Args&&... args) {
 }
 
 // A type of Orphanable with internal ref-counting.
-template <typename Child>
```

Comment:
Can you explain this change?

Reply:
This allows changing the behavior of `InternallyRefCounted<>` when the ref-count goes to 0, so that instead of deleting the object, we just call its dtor without freeing the memory.  This is useful for objects that are allocated on the arena.

I added this capability for `RefCounted<>` in #25848.  This basically just makes the same thing usable for `InternallyRefCounted<>`.

Reply:
Awesome! Thanks!